src/php/lib/Grpc/BaseStub.php

```diff
@@ -75,6 +75,48 @@ public function getTarget() {
   }
 
   /**
+   * @param $try_to_connect bool
+   * @return int The grpc connectivity state
+   */
+  public function getConnectivityState($try_to_connect = false) {
+    return $this->channel->getConnectivityState($try_to_connect);
+  }
+
+  /**
+   * @param $timeout in microseconds
+   * @return bool true if channel is ready
+   */
+  public function watchForReady($timeout) {
+    $new_state = $this->getConnectivityState(true);
+    if ($this->_checkConnectivityState($new_state)) {
+      return true;
+    }
+
+    $now = Timeval::now();
+    $delta = new Timeval($timeout);
+    $deadline = $now->add($delta);
```

Comment:
Currently, all the exposed PHP level API takes `timeout`, which got translated to `deadline` when passed to the C layer. (actually there are only 2 places like this, one when initiating a `Call` object and the other here). Is that OK to keep this?